app/src/molecules/GenericWizardTile/index.tsx

```diff
@@ -36,7 +36,7 @@ const GO_BACK_BUTTON_STYLE = css`
     opacity: 70%;
   }
 
-  @media ${ODD_MEDIA_QUERY_SPECS} {
+  @media ${RESPONSIVENESS.touchscreenMediaQuerySpecs} {
     font-weight: ${TYPOGRAPHY.fontWeightSemiBold};
     font-size: 1.375rem;
```

Comment:
not related to this PR

```suggestion
    font-size:${TYPOGRAPHY.fontSize22};
```